WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/DashboardPostsListCardCell.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  DashboardPostsListCardCell.swift
+//  WordPress
+//
+//  Created by Hassaan El-Garem on 11/04/2022.
+//  Copyright © 2022 WordPress. All rights reserved.
+//
+
```

Comment:
You can remove that. :D

Reply:
Done in 73ce7265434e678e9ce12118f1e72815403eb4f6 😅